src/parser/Parser.ts

```diff
@@ -2503,11 +2540,11 @@ function createReferences(): References {
 export interface References {
     assignmentStatements: AssignmentStatement[];
     classStatements: ClassStatement[];
-    namespaceStatements: NamespaceStatement[];
     functionStatements: FunctionStatement[];
     functionExpressions: FunctionExpression[];
```

Comment:
If you're ordering these alphabetically, might as well swap `functionStatements` and `functionExpressions` too.